src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -232,8 +231,8 @@ import { DateMixin } from '@/mixins'
 
 // Components
 import { AssociationDetails, DissolutionStatement } from '@/components/DefineDissolution'
-import { EffectiveDateTime, StaffPayment } from '@/components/common'
-import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+import { CourtOrderPoa, EffectiveDateTime } from '@/components'
```

Comment:
Example of loading shared components. In fact, this component doesn't know whether it's a shared component or not (which is good).

I didn't change over all the other shared imports, but at some point I think we should do that. It's not a huge improvement (so no rush) but it's a bit of cleanup.

The idea behind this: components should know as little as possible about "the world"; they should only know what they need to know. For example, this component knows that it needs these imports, but it shouldn't care where they come from.

Reply:
cool